package.json

```diff
@@ -30,14 +34,15 @@
     "agenda-rest": "cli.js"
   },
   "bugs": {
-    "url": "https://github.com/assister-ai/agenda-rest/issues"
+    "url": "https://github.com/agenda/agenda-rest/issues"
   },
-  "homepage": "https://github.com/assister-ai/agenda-rest#README",
+  "homepage": "https://github.com/agenda/agenda-rest#README",
   "devDependencies": {
+    "ava": "^0.25.0",
```

Comment:
Is there anything to test yet? :-)

Reply:
Nothing to test as there were no tests to begin with :|

Reply:
Yeah, that's something definitely needed. Guess tests would be easy to add when we settle on the http client in #6 since there are few endpoints.